corr-app/correlationDataHandler.py

```diff
@@ -0,0 +1,133 @@
+import pandas as pd
+import numpy as np
+import yfinance as yf
+
+class correlationDataHandler:
+    """
+    A data handler for the correlations
+    """
+    
+    def __init__(self, ticker_list: list):
+        self.ticker_list = ticker_list
+        
+    def build_frame(self, period) -> pd.DataFrame:
+        """
+        Get the values of interest from the list of securities from the user
+        """
+        
+        results = pd.DataFrame()
+        
+        for ticker in self.ticker_list:
+            df = yf.Ticker(ticker).history(period = period, interval = "1d")
+            # volume-weighted average price
+            df['vwap'] = (df["Volume"] * df["Close"]) / df["Volume"]
```

Comment:
Actually, I don't think we use vwap anyway. Can we just delete this line?

Reply:
Agreed